src/shell/command_line.cpp

```diff
@@ -350,12 +350,7 @@ uint16_t CommandLine::Get_arglength()
 		total_length += cmd.size() + 1;
 	}
 
-	if (total_length > UINT16_MAX) {
-		LOG_WARNING("CONFIG: Command line length too long, truncating");
-		total_length = UINT16_MAX;
-	}
-
-	return static_cast<uint16_t>(total_length);
+	return total_length;
```

Comment:
Was there any reason why total_length was limited to `UINT16_MAX` previously?

It might be good to explicitly cast the return type from `size_t` to `int`.

Reply:
I've just checked PVS-Studio and there are new warnings for this, so marking as resolved. I think that's because the entire `uint16_t` range fits within the signed `int32_t` range.